FaunaDB.Client/Client/FaunaClient.cs

```diff
@@ -32,13 +33,15 @@ public class FaunaClient
         /// <param name="endpoint">URL for the FaunaDB server. Defaults to "https://db.fauna.com:443"</param>
         /// <param name="timeout">Timeout for I/O operations. Defaults to 1 minute.</param>
         /// <param name="httpVersion">Version of http. Default value is HttpVersion.Version11, is you use .net core 3.0 and above you can enable http/2 support by passing HttpVersion.Version20</param>
+        /// <param name="dontCheckNewVersion">Don't check new NuGet package driver version. Default value is null (Check new version).</param>
         public FaunaClient(
             string secret,
             string endpoint = "https://db.fauna.com:443",
             TimeSpan? timeout = null,
             HttpClient httpClient = null,
-            Version httpVersion = null)
-            : this(CreateClient(secret, endpoint, timeout, httpClient, httpVersion))
+            Version httpVersion = null,
+            bool? dontCheckNewVersion = null)
```

Comment:
Can we have a default checkNewVersion parameter of `true` instead of negated one? In a similar way the `endpoint` parameter is made above.